awx/main/migrations/0088_v360_dashboard_optimizations.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.4 on 2019-09-10 21:30
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0087_v360_update_credential_injector_help_text'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='unifiedjob',
```

Comment:
The dashboard runs queries that filter or sort on all of these, but lots of _other_ critical part of our codebase do, too (like the task manager and the callback receiver).

Reply:
These indexes are where I saw the biggest speed improvement, and applying them to my 600K UJ table took less around 1 second.